helper_dart/lib/src/facet_list.dart

```diff
@@ -281,6 +296,15 @@ class _FacetList with DisposableMixin implements FacetList {
     });
   }
 
+  void _trackClickIfNeeded(String selection) {
+    _selections.first.then((selections) {
+      if (selections.contains(selection)) {
+        return;
+      }
+      eventTracker.trackClick(clickEventName, attribute, selection);
```

Comment:
```suggestion
      if (!selections.contains(selection)) {
        eventTracker.trackClick(clickEventName, attribute, selection);
      }
```
Maybe more readable like this 😄